{"name": "test_login_noExistUser_wrongPasswd[data0]", "status": "passed", "attachments": [{"name": "log", "source": "cde4c628-114e-42e0-92c5-98a76856b7ac-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "data", "value": "{'用户名': 'zhuhonghao', '密码': '', '预期': '请输入密码'}"}], "start": 1587375202332, "stop": 1587375203177, "uuid": "56627fa4-c5ed-4d5f-9445-02acccb47c43", "historyId": "265996dd5cab96e454bf9f896d2e1430", "testCaseId": "5a65c10c865bdd3f50e0040d88be9f16", "fullName": "TestCases.test_02_ptsdList.TestLogin#test_login_noExistUser_wrongPasswd", "labels": [{"name": "parentSuite", "value": "TestCases"}, {"name": "suite", "value": "test_02_ptsdList"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-O2A05I4"}, {"name": "thread", "value": "17492-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCases.test_02_ptsdList"}]}